paper/paper.md

```diff
@@ -0,0 +1,137 @@
+
+![Particles of two different systems in a simulation domain.\label{fig:systems}](systems.png){width=35%}
+
+Figure ? shows particles in a simulation domain. The black ones belong to system $(1)$ and the gray ones belong to system $(2)$.
+To calculate the forces acting on each particle, the particles need to interact with each other.
+In general, the force of a particle $a$ in system $(1)$ is calculated as following
+$$ f_a^{(1)} = \sum_{b \in \mathcal{N}^{(1)}} f_{ab}^{(1)} + \sum_{b \in \mathcal{N}^{(2)}} f_{ab}^{(2)} + \dots + \sum_{b\in \mathcal{N}^{(n)}}f_{ab}^{(n)}, $$
+
+where the superscripts denote the system, $\mathcal{N}$ is the set of neighboring particles of the corresponding system and $f_{ab}$ is the interaction force of particle $a$ and neighbor particle $b$. Each neighborhood is updated before the interaction with an efficient grid neighborhood search introduced in [@Ihmsen:2010].
+
+The interaction of those particle systems follows method specific rules. For example, the SPH method determines the acceleration of particle $a$ basically with the following interaction
+$$ \frac{d v_a}{d t} = -m_a \sum_b m_b \left( \frac{p_b}{\rho_a^2} + \frac{p_a}{\rho_a}^2 \right) \nabla_a W_{ab},$$
+where $v_a$ is the velocity of particle $a$ and $m_a$, $m_b$, $\rho_a$, $\rho_b$, $p_a$, $p_b$ is the mass, density and pressure of particle $a$ and $b$, respectively. The summation is over a weighting function also called kernel-function $W_{ab}$ depending on the relative distance of particle $a$ and $b$.
+
+Currently implemented systems:
+Fluid Systems
+- WeaklyCmpressibleSPHSystem (WCSPH): Standard SPH method originally developed by [@Monaghan:1977] to simulate astrophysics applications.
+- EntropicallyDampedSPHSystem (EDAC): As opposed to the WCSPH scheme, which uses an equation of state, this scheme uses a pressure evolution equation to calculate the pressure which is derived by [@Clausen:2013] and adapted to SPH by [@Ramachandran:2019].
+
+Strutrure System
+- TotalLagrangianSPHSystem (TLSPH): System to simulate elastic structure where the interaction is defined with stress tensors.
+
+Boundary Systems
+- BoundarySPHSystem with several boundary models where each model follows a different interaction rule.
+- OpenBoundarySPHSystem: System to simulate non-reflecting boundary conditions
+
+# Feature highlights
+
+Figure \autoref{fig:falling_sphere} illustrates an example of our simulation results.
+In this example, an elastic sphere modeled with Total Lagrangian SPH falls into a tank filled with water.
+
+![Elastic sphere falling into a tank filled with water.\label{fig:falling_sphere}](falling_sphere_combined_nonstick_4k_178.png){width=50%}
+
+Currently the following feature highlights are implemented
+- Weakly compressible SPH with density diffusion [@Antuono:2010]
+- Entropically damped artificial compressibility (EDAC) [@Ramachandran:2019]
+- Total Lagrangian SPH and fluid-structure interaction [@O_Connor:2021]
+- Discrete element method (DEM)
```

Comment:
@article{bicanic2004discrete,
  title={Discrete element methods},
  author={Bi{\'c}ani{\'c}, Nenad},
  journal={Encyclopedia of computational mechanics},
  year={2004},
  publisher={Wiley Online Library}
}


@article{cundall1979discrete,
  title={A discrete numerical model for granular assemblies},
  author={Cundall, Peter A and Strack, Otto DL},
  journal={geotechnique},
  volume={29},
  number={1},
  pages={47--65},
  year={1979},
  publisher={Thomas Telford Ltd}
}